imports/plugins/included/authnet/server/methods/authnet.js

```diff
@@ -90,7 +92,33 @@ Meteor.methods({
 
     const authnetService = getAuthnetService(getAccountOptions());
     const roundedAmount = parseFloat(amount.toFixed(2));
+    const capturedAmount = accounting.toFixed(amount, 2);
     let result;
+    if (capturedAmount === accounting.toFixed(0, 2)) {
+      try {
+        const captureResult = voidTransaction(transactionId,
+          authnetService
+        );
+        if (captureResult.responseCode[0] === "1") {
+          result = {
+            saved: true,
+            response: captureResult
+          };
+        } else {
+          result = {
+            saved: false,
+            error: captureResult
+          };
+        }
+      } catch (error) {
+        Logger.fatal(error);
```

Comment:
thought more than review:  should this be a `warn` since it's not actually a `fatal` error because we're capturing it?